lib/learn_to_rank/reranker.rb

```diff
@@ -20,6 +21,7 @@ def rerank(query: "", es_results: [])
       reorder_results(es_results, new_scores)
     rescue StandardError => e
       GovukError.notify(e, extra: { query: query })
```

Comment:
This `GovukError.notify` needs to go away now, or we'll notify entry twice.

Reply:
🤔 I think the other cases where we call `GovukError.notify` in `Ranker` are also in rescue blocks, so this call in the `rescue StandardError` block won't get called, but I'd still like to keep it as a catch all for those that don't get rescued.

Reply:
Yeah, I suspect this block is redundant, but in case it's not we now have

```
rescue StandardError => e
  GovukError.notify(e, extra: { query: query })
  report_error(e, extra: { query: query })
```

And `report_error` also calls `GovukError.notify`

Reply:
Ah sorry you are totally right, I've removed that call.